### PR TITLE
Added yelp loader and transformer

### PR DESCRIPTION
**Pull request by Tekrus**

### CODE REVIEW

File: model/Graph-conv-context-aware-idea-2021-wip/Data/yelp/yelp_train_test.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            categories_dict[category].append(1)
+        else:
+            categories_dict[category].append(0)
+
+for key, value in tqdm(categories_dict.items()):
+    df[key] = value
+
+df.drop('categories', inplace=True, axis=1)
+
+print(len(categories_dict))
```
Comment (Eduardsen) [inline model/Graph-conv-context-aware-idea-2021-wip/Data/yelp/yelp_train_test.py]: "jeg fjerner lige mine prints" 👺

Comment (Tekrus) [reply model/Graph-conv-context-aware-idea-2021-wip/Data/yelp/yelp_train_test.py]: 👺👺👺👺

Comment (andpaacoke) [reply model/Graph-conv-context-aware-idea-2021-wip/Data/yelp/yelp_train_test.py]: 👹

Comment (Tekrus) [inline model/Graph-conv-context-aware-idea-2021-wip/Data/yelp/yelp_train_test.py]:
```suggestion
```

**Review by Eduardsen — APPROVED** [review]

**Review by andpaacoke — APPROVED** [review]

**Pull request merged**